init.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 export PYTHONPATH=.
-export SOF_DATA=~/astar/METIS_Simulations/ESO/output
+export SOF=~/astar/METIS_Pipeline_Test_Data/metis_sim_small_1/sof
+export SOF_DATA=~/astar/METIS_Pipeline_Test_Data/metis_sim_small_1/data
 export PYESOREX_PLUGIN_DIR=~/astar/METIS_Pipeline/metisp/pyrecipes/
+export PYESOREX_OUTPUT_DIR=~/astar/METIS_Pipeline_Test_Data/metis_sim_small_1/data
```

Comment:
I think we should remove this file, or at least use the same paths as we use in the Dockerfile. But that is not really relevant to this PR though

Reply:
Fixed in #64